### PR TITLE
Carol postedhouse

### PR DESCRIPTION
**Pull request by imcarolll**
completed follow and unfollow button on other profile screen

### CODE REVIEW

File: src/house/services/auth-thunks.js [hunk 1]
```diff
@@ -33,6 +33,14 @@ export const updateUserThunk = createAsyncThunk(
   }
 );
 
+// export const followThunk = createAsyncThunk(
```
Comment (XinyaChen0216) [inline src/house/services/auth-thunks.js]: nit: remove all the `console.log` and commented code, thanks

**Review by XinyaChen0216 — APPROVED** [review]
other than the nit as commented, lgtm

File: src/house/houses/houseList.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ const HouseList = () => {
   } else generalHouses = totalHouses;
   useEffect(() => {
     dispatch(findHousesThunk());
-    setTimeout(() => dispatch(findPublicHousesThunk()), 500);
+    //setTimeout(() => dispatch(findPublicHousesThunk()), 500);
```
Comment (XinyaChen0216) [inline src/house/houses/houseList.js]: nit: we might need it line uncommented in main, thanks

File: src/house/services/auth-thunks.js (hunk 1 above)
Comment (imcarolll) [reply src/house/services/auth-thunks.js:36]: updated and will include in my next pull request

Comment (imcarolll) [reply src/house/services/auth-thunks.js]: addressed

File: src/house/houses/houseList.js (hunk 2 above)
Comment (imcarolll) [reply src/house/houses/houseList.js]: addressed

**Pull request merged**